cpp/src/parquet/encryption/internal_file_decryptor.cc

```diff
@@ -64,16 +64,6 @@ InternalFileDecryptor::InternalFileDecryptor(FileDecryptionProperties* propertie
   properties_->set_utilized();
 }
 
-void InternalFileDecryptor::WipeOutDecryptionKeys() {
-  std::lock_guard<std::mutex> lock(mutex_);
-  properties_->WipeOutDecryptionKeys();
```

Comment:
`properties_->WipeOutDecryptionKeys` will clear the private key, we should keep it no?
(even though it's currently not reliable: see https://github.com/apache/arrow/issues/31603)

Reply:
restored